src/pointquery.js

```diff
@@ -21,6 +21,20 @@ mora.createMap = async function(config) {
     mora.emit('query-results', result);
   });
 
+  mora.on('query-results', function(res) {
+    if (res.dichtstbijzijnd_adres !== null) {
+      let adres = res.dichtstbijzijnd_adres;
+      let display_string = `${adres.openbare_ruimte} ${adres.huisnummer}
+${adres.huisletter ? adres.huisletter : ''}
+${adres.huisnummer_toevoeging ? '-'+adres.huisnummer_toevoeging : ''}
+, ${adres.postcode} ${adres.woonplaats}`
+      document.getElementById('nlmaps-geocoder-control-input').value = display_string;
```

Comment:
Dit is wel een heel directe koppeling met de nlmaps internals. Er is nu zeker nog geen API in nlmaps om dit door te geven?

Reply:
Bedoel je specifiek het zetten van #nlmaps-geocoder-control-input.value? Ja, dat is wel zo. We hebben nog niets hiervoor regegeld omdat de design filosofie van nlmaps is om het vrij low-level te houden en niet dit soort koppelingen voor de gebruiker te maken. Bovendien is het samenstellen van de string afhankelijk van de amsterdam bag api. Maar je zou kunnen verwachten dat er vervolgens een setter is om aan te roepen ipv direct de value veranderen. Bedoel je dat?

Reply:
Ik bedoelde vooral dat je het ID gebruikt van een element, maar ook inderdaad wel dat je direct de value op dat element zet.